pkgs/os-specific/linux/libselinux/fix-static-build.patch

```diff
@@ -0,0 +1,105 @@
+diff --color -ur a/src/Makefile b/src/Makefile
+--- a/src/Makefile	2019-11-28 13:46:48.000000000 +0100
++++ b/src/Makefile	2021-08-21 20:18:42.554513955 +0200
+@@ -32,7 +32,7 @@
+ endif
+ 
+ LIBA=libselinux.a 
+-TARGET=libselinux.so
++# TARGET=libselinux.so
```

Comment:
```suggestion
```

When rebasing this patch in the future it will be easier when the line is just deleted.